lib/create_household/view/create_household.dart

```diff
@@ -0,0 +1,200 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_form_builder/flutter_form_builder.dart';
+import 'package:form_builder_validators/form_builder_validators.dart';
+import 'package:geolocator/geolocator.dart';
+import 'package:gibsonify_api/gibsonify_api.dart';
+import 'package:gibsonify_repository/gibsonify_repository.dart';
+
+import '../bloc/create_household_bloc.dart';
```

Comment:
can we have absolute paths just to keep it consistent

Reply:
Done